helm/README.md

```diff
@@ -120,95 +120,182 @@ For Helm chart, you'll need to craft a `values.yaml`.
+       dbType *
       </td>
       <td>
        Type of database server can be used for configuring the Bold BI.<br/><br />The following DB types are accepted:<br />1. mssql – Microsoft SQL Server/Azure SQL Database<br />2. postgresql – PostgreSQL Server<br />3. mysql – MySQL/MariaDB Server
       </td>
     </tr>
     <tr>
       <td>
-       BOLD_SERVICES_DB_HOST *
+       dbHost *
       </td>
       <td>
        Name of the Database Server
       </td>
     </tr>
     <tr>
       <td>
-       BOLD_SERVICES_DB_PORT
+       dbPort
       </td>
       <td>
        The system will use the following default port numbers based on the database server type.<br />PostgrSQL – 5234<br />MySQL -3306<br /><br />Please specify the port number for your database server if it is configured on a different port.<br /><br />For MS SQL Server, this parameter is not necessary.
       </td>
     </tr>
     <tr>
       <td>
-       BOLD_SERVICES_DB_USER *
+       dbUser *
       </td>
       <td>
        Username for the database server<br /><br />Please refer to [this documentation](https://help.boldbi.com/embedded-bi/faq/what-are-the-database-permissions-required-to-set-up-bold-bi-embedded/) for information on the user's permissions.
       </td>
     </tr>
     <tr>
       <td>
-       BOLD_SERVICES_DB_PASSWORD *
+       dbPassword *
       </td>
       <td>
        The database user's password
       </td>
     </tr>
     <tr>
       <td>
-       BOLD_SERVICES_DB_NAME
+       dbName
       </td>
       <td>
        If the database name is not specified, the system will create a new database called bold services.<br /><br />If you specify a database name, it should already exist on the server.
       </td>
     </tr>
     <tr>
       <td>
-       BOLD_SERVICES_POSTGRESQL_MAINTENANCE_DB
+       maintenanceDB
       </td>
       <td>
        For PostgreSQL DB Servers, this is an optional parameter.<br />The system will use the database name `postgres` by default.<br />If your database server uses a different default database, please provide it here.
       </td>
     </tr>
     <tr>
       <td>
-       BOLD_SERVICES_DB_ADDITIONAL_PARAMETERS
+       dbAdditionalParameters
       </td>
       <td>
        If your database server requires additional connection string parameters, include them here.<br /><br />Connection string parameters can be found in the official document.<br />My SQL: https://dev.mysql.com/doc/connector-net/en/connector-net-8-0-connection-options.html<br />PostgreSQL: https://www.npgsql.org/doc/connection-string-parameters.html<br />MS SQL: https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnection.connectionstring<br /><br /><b>Note:</b> A semicolon(;) should be used to separate multiple parameters.
       </td>
     </tr>
+</table>
+<br/>
+
+## Environment variables for configuring Branding in backend
+
+The below Environment variables are optional. If not provided Bold BI will take the default configured values.
```

Comment:
The following environment variables are optional. If they are not provided, Bold BI will use the default configured values.

Reply:
concern addressed